modules/sms_user/config/install/sms_user.settings.yml

```diff
@@ -1,6 +1,15 @@
 active_hours:
   status: false
   ranges: []
-registration_enabled: false
-allow_password: false
-new_account_message: ''
+account_registration:
+  all_unknown_numbers:
+    status: false
+    reply_message: ''
+  formatted:
+    status: false
+    incoming_messages: []
+    activation_email: true
+    reply:
+      status: false
+      message: ''
+      message_failure: ''
```

Comment:
`all_unknown_numbers` is not sufficiently explanatory. It still doesn't tell what it does. Same with `formatted` in the examples you have given. The lower rung names can be left as is but would only make meaning if the other two are self-explanatory.